nx-X11/programs/Xserver/os/osdep.h

```diff
@@ -61,15 +61,9 @@ SOFTWARE.
 
 #include <X11/Xdmcp.h>
 
-#ifndef sgi	    /* SGI defines OPEN_MAX in a useless way */
 #ifndef X_NOT_POSIX
 #ifdef _POSIX_SOURCE
 #include <limits.h>
-#else
```

Comment:
This else is for `_POSIX_SOURCE`, you're a few layers down into `if` macros here!